src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs

```diff
@@ -124,18 +124,22 @@ await _notification.Clients.Clients(NotificationHub.AdminConnectionIds)
 
 
                 var customization = await _customizationSettings.GetSettingsAsync();
-                // Get the Content
-                var plexContent = _plex.GetAll().Include(x => x.Episodes).AsNoTracking();
-                var embyContent = _emby.GetAll().Include(x => x.Episodes).AsNoTracking();
-                var jellyfinContent = _jellyfin.GetAll().Include(x => x.Episodes).AsNoTracking();
-                var lidarrContent = _lidarrAlbumRepository.GetAll().AsNoTracking().ToList().Where(x => x.FullyAvailable);
+                // MOVIES
```

Comment:
It's not really just movies, it's Movies + Shows and Episodes (due to the `.include`)

Reply:
You're right. This part should change a lot by the end of this PR anyway.